src/cmd/uds.go

```diff
@@ -98,17 +125,21 @@ var removeCmd = &cobra.Command{
 	Aliases: []string{"r"},
 	Args:    cobra.ExactArgs(1),
 	Short:   lang.CmdBundleRemoveShort,
-	Run: func(_ *cobra.Command, args []string) {
+	RunE: func(_ *cobra.Command, args []string) error {
 		bundleCfg.RemoveOpts.Source = args[0]
 		configureZarf()
 
-		bndlClient := bundle.NewOrDie(&bundleCfg)
+		bndlClient, err := bundle.New(&bundleCfg)
+		if err != nil {
+			return err
+		}
 		defer bndlClient.ClearPaths()
 
 		if err := bndlClient.Remove(); err != nil {
 			bndlClient.ClearPaths()
-			message.Fatalf(err, "Failed to remove bundle: %s", err.Error())
+			return fmt.Errorf("Failed to remove bundle: %s", err.Error())
```

Comment:
Let's make sure errors start with a lowercase letter, will make a follow up ticket to make the pre-commit check this